rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -1405,6 +1428,7 @@ class DebruijnInterpreter[M[_], F[_]](
       "size"        -> size,
       "length"      -> length,
       "slice"       -> slice,
+      "take"        -> take,
```

Comment:
It looks like the performance hit of using a ListOps version of take is so big that integration tests fail due to timeouts. Also had to increase the timeout for the `PoSSpec`.

Reply:
Does this mean that the Rholang implementation of ListOps is not really usable?

Do we have a rule about putting list operations in the list methods or in the ListOps contract?

Reply:
Let me take that back. Actually integration tests failing because of timeouts is probably also due to the overall performance decrease that comes with the new PoS logic.
However `PoSSpec` passed on my laptop with a 120s timeout before the change to `ListOps!take`. Which wasn't the case afterwards.

There was a discussion between Joseph, Artur and Mike in #rholang and I think everyone agreed that methods are not the way to go.

Reply:
@tymm That was a slightly different conversation. It was a debate to determine whether the cryptographic functions should be methods or system processes. I would go with adding `take` as a method alongside `slice` and `nth`. It's bound to be more performant and won't introduce unnecessary bloat in the tuplespace. It's also syntactically cleaner.

Reply:
@JosDenmark 's argument that "take" is not a higher-order function so it should be a method sounds fair to me